resources/js/Components/Form/DmEntryCreateForm.tsx

```diff
@@ -118,212 +128,288 @@ const DmEntryCreateForm = ({
+                <StyledGrid item xs={12} md={type === 'Edit' ? 12 : 2}>
+                    <InputLabel shrink>Assigned Character</InputLabel>
+                    <StyledSelect
+                        margin='normal'
+                        fullWidth
+                        size='small'
+                        id='character_id'
+                        select
+                        name='Assigned Character'
+                        SelectProps={{
+                            MenuProps: {PaperProps: {sx: {maxHeight: 300}}},
+                        }}
+                        value={data.character_id}
+                        onChange={(e) =>
+                            setData('character_id', parseInt(e.target.value))
+                        }>
+                        {characters?.map((option: {id: number; name: string}) => (
+                            <MenuItem key={option.id} value={option.id}>
+                                {option.name}
+                            </MenuItem>
+                        ))}
+                    </StyledSelect>
+                    {errors?.character_id && <ErrorText message={errors?.character_id} />}
+                </StyledGrid>
+                <StyledGrid item xs={12}>
+                    <TextField
+                        margin='normal'
+                        fullWidth
+                        id='notes'
+                        label='Notes'
+                        name='Notes'
+                        value={data.notes}
+                        onChange={(e) => setData('notes', e.target.value)}
                     />
-                    {activeStep === 1 && data.items.length > 0 && (
-                        <StyledItemsFooter container spacing={1}>
-                            <StyledGrid item xs={12}>
-                                <Typography variant='body2'>New Magic Items</Typography>
-                            </StyledGrid>
-                            {data.items.map((item) => (
-                                <StyledGrid item xs='auto' key={item.name}>
-                                    <Chip
-                                        variant='outlined'
-                                        label={item.name}
-                                        sx={{color: '#86B8F4', borderColor: '#86B8F4'}}
-                                        onDelete={() => handleDeleteItem(item)}
-                                    />
-                                </StyledGrid>
-                            ))}
-                        </StyledItemsFooter>
-                    )}
-                </>
+                    {errors?.notes && <ErrorText message={errors?.notes} />}
+                </StyledGrid>
+            </Grid>
+        </>
+    )
+
+    const stepTwoContent = (
+        <>
+            <ItemCreateForm
+                type='Create'
+                onCloseDrawer={onCloseDrawer}
+                cancelFormButton={
```

Comment:
is there a reason to pass this down from here? it does not depend anything from `DMEntryCreateForm`

Reply:
Are you talking about the `onCloseDrawer` function?

Reply:
nope, the `cancelFormButton`, there is no need of passing down since it does not depend anything from the parent component

Reply:
I see, I'll remove it.